docs/deployment/rds-s3/README.md

```diff
@@ -58,23 +58,17 @@ To install for both RDS and S3 complete all the below steps.
 
 1. Clone the `awslabs/kubeflow-manifest` repo, `kubeflow/manifests` repo and checkout the release branches.
 
-   - Substitute the value for `KUBEFLOW_RELEASE_VERSION`(e.g. v1.4.1) and `AWS_RELEASE_VERSION`(e.g. v1.4.1-aws-b1.0.0) with the tag or branch you want to use below. Read more about [releases and versioning](../../community/releases.md#releases-and-versioning) policy if you are unsure about what these values should be.
-     ```
-     export KUBEFLOW_RELEASE_VERSION=<>
-     export AWS_RELEASE_VERSION=<>
-     git clone https://github.com/awslabs/kubeflow-manifests.git && cd kubeflow-manifests
-     git checkout ${AWS_RELEASE_VERSION}
-     git clone --branch ${KUBEFLOW_RELEASE_VERSION} https://github.com/kubeflow/manifests.git upstream
-     ```
```

Comment:
assuming this is mentioned elsewhere

Reply:
Waiting on the common pre-reqs pr to get merged